### DISCUSSION TITLE
How to improve night detection?

### DISCUSSION THREAD

**Discussion by joseror** [blakeblackshear/frigate#6249] [Ask A Question]
I have a driveway with lighs that point directly at the camera. If nothing is happening it looks like this:

![driveway](https://user-images.githubusercontent.com/49996350/234315819-c5099ee5-b8d6-4616-8dc2-ab50abb2de6e.png)

It does detection motion, but when a car drives by the lights keep it too light for the camera to be able to identify objects:

![car](https://user-images.githubusercontent.com/49996350/234316008-ac6480ff-8fad-4cd0-86ee-d7a1e5431b61.png)
![car2](https://user-images.githubusercontent.com/49996350/234316015-250ec703-86bb-4cf0-8c9d-1c3386dc70f1.png)

Would improve_contrast make any different here? Or do you have any ideas on how I could improve night performance? For reference, here's a day view:

![image](https://user-images.githubusercontent.com/49996350/234317206-790fdd3b-e930-44ed-a8f5-e0fe03efa00e.png)

**Comment by NickM-27** [comment]
This seems like the only way to improve would be on the camera itself whether that means enabling wide dynamic range (WDR), glare compensation in the night profile, etc.

**Reply by joseror** [reply]
> This seems like the only way to improve would be on the camera itself whether that means enabling wide dynamic range (WDR), glare compensation in the night profile, etc.

This is an analog camera, so I dont even have an option for a separate night profile in the NVR.

**Reply by NickM-27** [reply]
oh, then maybe get a better camera. otherwise you could see if there are some ffmpeg args to adjust the image, but that usually a camera setting

**Comment by blakeblackshear** [comment]
As a human watching that video feed, I'm not sure I could identify a car at night. I would be shocked if you could get an kind of reliable detection from that camera at night.
